### PR TITLE
[FAN-173] Other profile page

### PR DESCRIPTION
**Pull request by Antroll**
https://fangible.atlassian.net/browse/FAN-173

### CODE REVIEW

File: src/modules/brand/screens/MyBrand/MyBrand.tsx [hunk 1]
```diff
@@ -107,51 +108,55 @@ export const MyBrand = () => {
...
-                }}
-                toSale={RoutesConfiguration.PublishNft.generatePath(
-                  item.contractaddress,
-                  item.id,
-                )}
-              />
-            </Grid>
-          ))}
+          {items?.map((item: any) => {
+            const shortAddr = address ? truncateWalletAddr(address) : 'Unknown';
```
Comment (BigBadAlien) [inline src/modules/brand/screens/MyBrand/MyBrand.tsx]: t('unknown')

File: src/modules/overview/components/Products/Products.tsx [hunk 2]
```diff
@@ -125,7 +129,21 @@ export const Products = ({ ...sectionProps }: ISectionProps) => {
           objectFit: 'scale-down',
           loading: 'lazy',
         }}
-        ProfileInfoProps={item.ProfileInfoProps}
+        profileInfo={
+          <ProfileInfo
+            subTitle="Owner"
```
Comment (BigBadAlien) [inline src/modules/overview/components/Products/Products.tsx:134]: t

File: src/modules/profile/actions/fetchProfileInfo.ts [hunk 3]
```diff
@@ -1,40 +1,66 @@
...
+      };
 
-        return request;
-      },
-      getData: mapProfileInfo,
+      return request;
+    },
+    getData: data => {
+      if (data.code !== 1) {
+        console.error(data?.msg ?? 'Unexpected error');
```
Comment (BigBadAlien) [inline src/modules/profile/actions/fetchProfileInfo.ts:60]: replace console by throw

**Review by BigBadAlien — APPROVED** [review]

File: src/modules/profile/actions/fetchProfileInfo.ts (hunk 3 above)
Comment (Antroll) [reply src/modules/profile/actions/fetchProfileInfo.ts:60]: In this case, your option is not suitable.

**Pull request merged**